controllers/util/mw_util.go

```diff
@@ -285,13 +285,19 @@ func (mwu *MWUtil) CreateOrUpdateNamespaceManifest(
 		return err
 	}
 
-	labels := map[string]string{}
 	manifests := []ocmworkv1.Manifest{
 		*manifest,
 	}
 
 	mwName := fmt.Sprintf(ManifestWorkNameFormat, name, namespaceName, MWTypeNS)
-	manifestWork := mwu.newManifestWork(mwName, managedClusterNamespace, labels, manifests, annotations)
+	manifestWork := mwu.newManifestWork(
+		mwName,
+		managedClusterNamespace,
+		map[string]string{
+			"cluster.open-cluster-management.io/backup": "",
```

Comment:
(nit) Use rmnutil.OCMBackupLabelKey instead of the label directly (makes it easier to grep code later). We also seem to pass a value of "resource" (which maybe superfluous) in other places `OCMBackupLabelValue string = "resource"` is that needed?

Reply:
Good point. It is not needed, but it is good to be consistent.  I just copied what we had in the VRG MW.